### DISCUSSION TITLE
floating-point binary to decimal

### DISCUSSION THREAD

**Discussion by yusuforzibekov** [community/community#71123] [Programming Help]
### Body

Hello, everyone! Does anyone know how to convert a floating-point number to a decimal? Could you write a function that can perform this task?

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by ebubekirgungor** [comment]
Hello, in which programming language do you want?

**Reply by yusuforzibekov** [reply]
Python

**Comment by Piero24** [comment]
You can convert a floating-point number to a decimal using the decimal module:

```python
from decimal import Decimal

floating_point_number = 3.14159
decimal_number = Decimal(str(floating_point_number))
print(decimal_number)
```
